lib/kinetic_cafe/error.rb

```diff
@@ -176,6 +176,10 @@ def default_status
       defined?(Rack::Utils) && :bad_request || 400
     end
 
+    def default_level
+      ::Logger::Severity::ERROR
```

Comment:
If we get rid of the dependency on `Logger` here as a hard requirement, then this can just be `:error`. This is better for us because if we are setting an error level that matches `Logger::Severity` names, we can *also* use that in [`kinetic_cafe_error_log_error`](https://github.com/KineticCafe/kinetic_cafe_error/pull/8/files#diff-750e90522343a2f1f01a2ef5ed401428L107). Namely, that can become:

```ruby
def kinetic_cafe_error_log_error(error)
  locale = self.class.kinetic_cafe_error_handler_log_locale
  Rails.logger.send(error.level, error.message(locale))

  return unless error.cause
 
  Rails.logger.send(
    error.level,
    t(
      'kinetic_cafe_error.cause',
      message: error.cause.message,
      locale: locale
    )
  )
end
```

I’m not familiar enough with `Rails.logger` to know how to send the error level as a parameter, but this seems clean enough and encourages overall readability.